src/msw/uilocale.cpp

```diff
@@ -437,7 +453,18 @@ class wxUILocaleImplName : public wxUILocaleImpl
             }
             else
             {
-                wxLogLastError(wxT("GetUserPreferredUILanguages"));
+                // Use the default user locale for Windows 7 resp Windows 8.x and below
+                wchar_t buf[LOCALE_NAME_MAX_LENGTH];
+                if (!ms_GetUserDefaultLocaleName(buf, LOCALE_NAME_MAX_LENGTH))
+                {
+                    const wxString language(buf);
+                    preferred.push_back(language);
```

Comment:
Do we really need a temporary variable? I.e. why not just
```suggestion
                    preferred.push_back(buf);
```